packages/flutter/lib/src/rendering/animated_size.dart

```diff
@@ -16,32 +16,37 @@ enum RenderAnimatedSizeState {
   /// The initial state, when we do not yet know what the starting and target
   /// sizes are to animate.
   ///
-  /// Next possible state is [stable].
+  /// The next state is [stable].
   start,
 
   /// At this state the child's size is assumed to be stable and we are either
   /// animating, or waiting for the child's size to change.
   ///
-  /// Next possible state is [changed].
+  /// If the child's size changes, the state will become [changed]. Otherwise,
+  /// it remains [stable].
   stable,
 
   /// At this state we know that the child has changed once after being assumed
   /// [stable].
   ///
-  /// Next possible states are:
+  /// The next state will be one of:
   ///
-  /// - [stable] - if the child's size stabilized immediately, this is a signal
-  /// for us to begin animating the size towards the child's new size.
-  /// - [unstable] - if the child's size continues to change, we assume it is
-  /// not stable and enter the [unstable] state.
+  /// - [stable] if the child's size stabilized immediately. This is a signal
```

Comment:
nit: I believe we use `*` in most of the other lists, such as "See also:" lists.

Reply:
fixed